src/components/RocketsContainer.js

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getRocket } from '../redux/rockets/rockets';
+import Rocket from './Rocket';
+
+const RocketsContainer = () => {
+  const rockets = useSelector((state) => state.rocket);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    if (!rockets.length) {
+      dispatch(getRocket());
+    }
+  }, [dispatch, rockets.length]);
```

Comment:
[OPTIONAL] I don't think you need this conditional statement. If you want to fetch data only once when the page is loaded  I think you should just leave the dependency array empty.

Reply:
We tried first leaving it empty but eslint error showed up. I emptied it and add eslint-disable line for that case.